clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -904,6 +904,7 @@ static UpdateStoreQueryParams getUpdateStoreQueryParams(CommandLine cmd) {
         p -> params.setNativeReplicationSourceFabric(p),
         argSet);
     booleanParam(cmd, Arg.ACTIVE_ACTIVE_REPLICATION_ENABLED, p -> params.setActiveActiveReplicationEnabled(p), argSet);
+    booleanParam(cmd, Arg.BEFORE_IMAGE_ENABLED, p -> params.setBeforeImageEnabled(p), argSet);
```

Comment:
In the discussion, we talk about change log with or without before image.
What is the plan to enable these combinations?

Reply:
After discussed with Zac. We discard the previous enabling store config change, as his PR contains what we need: https://github.com/linkedin/venice/pull/57. 

I reuse this PR for adding change capture topic switch control message event.